metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -267,6 +275,18 @@ def create(
         obj.is_project,
     )
 
+    # parse key value pairs into a dict for incident.io metadata if given
+    parsed_metadata = None
+    if incident_io_metadata is not None:
+        parsed_metadata = {}
+        for kv in incident_io_metadata:
+            key, value = kv.split("=", 1)
```

Comment:
minor nit - not a blocker for release but we can move this logic to argo_workflows.py